monai/networks/nets/vitautoenc.py

```diff
@@ -46,21 +46,25 @@ def __init__(
         pos_embed: str = "conv",
         dropout_rate: float = 0.0,
         spatial_dims: int = 3,
+        qkv_bias: bool = False,
+        save_attn: bool = False,
     ) -> None:
         """
         Args:
-            in_channels: dimension of input channels or the number of channels for input
-            img_size: dimension of input image.
-            patch_size: dimension of patch size.
-            hidden_size: dimension of hidden layer.
-            out_channels: number of output channels.
-            deconv_chns: number of channels for the deconvolution layers.
-            mlp_dim: dimension of feedforward layer.
-            num_layers: number of transformer blocks.
-            num_heads: number of attention heads.
-            pos_embed: position embedding layer type.
-            dropout_rate: faction of the input units to drop.
-            spatial_dims: number of spatial dimensions.
+            in_channels (int): dimension of input channels or the number of channels for input.
+            img_size (Sequence[int] | int): dimension of input image.
+            patch_size (Sequence[int] | int): dimension of patch size
+            out_channels (int, optional):  number of output channels. Defaults to 1.
```

Comment:
Hi @a-parida12, we haven't been including types in the docstrings for parameters since they're included in the declaration. Stylistically it would match better with MONAI not to have them. Also parameters like `out_channels` are not considered optional, ie. they can be `None`. You can optionally not provide a value for them when calling since they have defaults, but this is different from the parameter itself being optional. Thanks!

Reply:
@ericspod thanks for pointing it out to me! I will remove the types from the docstring just to be consistent throughout MonAI